### ISSUE TITLE
Feature - user pref to specify initial window size on record/play

### ISSUE THREAD

**Issue by coinzdude** [SeleniumHQ/selenium-ide#1611]
New user preference for config for initial record and playback window size

**Comment by toddtarsi** [comment]
@coinzdude - Hey I know you! Project level? For internal side, i need to clean up the dimension tracking.

**Comment by coinzdude** [comment]
Yeah, like you reminded me here (and I think I actually did that?! lol bad memory cells!) we already have some persistence memory by stuffing window sizes into preferences on close.  I think window start might be 'special' though in that you want more fixed control over the initial launch window size on tests globally while working.  I'll kick it around and see what might work best in practice.
